README.md

```diff
@@ -1 +1,74 @@
-# golang-training-Theater
\ No newline at end of file
+# golang-training-Theater
+
+## Description
+
+##### Full Stack Application for storing theater
```

Comment:
why you named it like "Full Stack application"?

Reply:
Done)